syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "google/protobuf/timestamp.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message AudioFrame {
  google.protobuf.Timestamp device_timestamp = 1;
  repeated float data = 2;
}
